mmv1/third_party/terraform/services/bigtable/resource_bigtable_instance.go

```diff
@@ -63,6 +63,10 @@ func ResourceBigtableInstance() *schema.Resource {
 				Description: `A block of cluster configuration options. This can be specified at least once.`,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"state": {
```

Comment:
We'll want to add a `Description` to this field for wrapper tools (i.e. KCC) and a website entry under the Attributes reference, the path of the field being `cluster.0.state`: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/r/bigtable_instance.html.markdown#attributes-reference

Reply:
Fixed